das/local_disk_das.go

```diff
@@ -90,12 +116,15 @@ func (das *LocalDiskDAS) Store(ctx context.Context, message []byte, timeout uint
 		return nil, err
 	}
 
+	c.KeysetHash = das.keysetHash
+
 	return c, nil
 }
 
 func (das *LocalDiskDAS) Retrieve(ctx context.Context, certBytes []byte) ([]byte, error) {
 	cert, err := arbstate.DeserializeDASCertFrom(bytes.NewReader(certBytes))
 	if err != nil {
+		fmt.Println("======> AAA")
 		return nil, err
 	}
 
```

Comment:
Just confirming my understanding; we don't set the keyset hash on the returned certs here because we are already checking in the aggregator that the returned responses are correctly signed by that DAS's public key, which is the sole contents of the keyset of this DAS.